knative-operator/pkg/webhook/knativeeventing/webhook_mutating.go

```diff
@@ -34,7 +33,10 @@ func (v *Configurator) Handle(ctx context.Context, req admission.Request) admiss
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
-	common.MutateEventing(ke)
+	// Unset the entire registry section. We used to override it anyway, so there can't
+	// be any userdata in there.
+	// TODO: Remove in the 1.21 release to potentially make this usable for users.
```

Comment:
Should we track that as a JIRI to noforget it? :see_no_evil:

Reply:
I guess we can.